osp/models/form_feedback.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.db import models
+
+from osp.models.form import Form
+from osp.models.answer import Answer
+from osp.utils import choices
+
+class FormFeedback(models.Model):
+
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    ) 
+    form = models.ForeignKey(
+        Form,
+        on_delete=models.CASCADE
+    )
+    answers = models.ManyToManyField(
+        to=Answer,
+        default=None,
+        blank=True,
+        related_name='form_answers'
```

Comment:
Just `forms`
otherwise from Answer object it would look like:
`answer_obj.form_answers.all()` which isn't helpful